examples/datasets_io/benchmark_lazy_eager_loading.py

```diff
@@ -335,10 +336,9 @@ def run_training(model, dataloader, loss, optimizer, n_epochs=1, cuda=False):
         num_workers=num_workers,
         worker_init_fn=None,
     )
-
     # Instantiate model and optimizer
-    n_channels = len(dataset.datasets[0].windows.ch_names)
-    n_times = len(dataset.datasets[0].windows.times)
+    n_channels = len(dataset[0][0].shape[0])
+    n_times = len(dataset[0][0].shape[1])
```

Comment:
I think the `len` causes this: `TypeError("object of type 'int' has no len()")`

maybe:
```
n_channels = dataset[0][0].shape[0]
n_times = dataset[0][0].shape[1]
```